src/main/xerus/monstercat/Settings.kt

```diff
@@ -42,6 +42,36 @@ object Settings : SettingsNode("xerus/monsterutilities") {
 	
 	val FILENAMEPATTERN = create("updatePattern", "MonsterUtilities-%version%.jar")
 	
+	enum class ConnectionSpeed(val maxConnections: Int, val display: String) {
+		DIALUP(5, "Dial-up (100 kb/s)"),
+		ADSL(30, "ADSL / 3G (10 Mb/s)"),
+		CABLE(150, "Cable / 4G (100 Mb/s)"),
+		FIBER(300, "Fiber (300+ Mb/s)");
+
+		override fun toString(): String = display
+		
+		companion object {
+			fun findFromValue(maxConnections: Int): ConnectionSpeed {
+				ConnectionSpeed.values().forEach {
```

Comment:
here the same as in the other PR, use `find` and then `?:`

Reply:
5263780293018ae18f658f88680131dc8732fd15